modules/browser/ExternalBrowserCapabilities.js

```diff
@@ -0,0 +1,65 @@
+import BrowserCapabilities from './BrowserCapabilities';
+
+/**
+ * API for checking the capabilities for Jitsi Meet for the current or passed
```

Comment:
Is this the entry point for the new bundle? Can we add a note on what the purpose is?

Reply:
The entry point is index.js and there is a comment. See https://github.com/jitsi/lib-jitsi-meet/blob/423de642b682c5c24b344662970731d001261e16/modules/browser/index.js#L4

Reply:
The documentation in this file doesn't make it clear why it exists (at least I don't understand it after reading it). It's not obvious why it just re-exposes part of an already available API and nothing else. But perhaps this is a common practice that I'm not familiar with.

Reply:
My idea behind it is the following:
1) The index.js is the only entry point for the module.
2) Everything in the module should be used trough the index.js

Most of the features in Jitsi Meet are implemented this way. I'm not sure if we have it somewhere in a code style guide.